src/commands/lambda/constants.ts

```diff
@@ -96,6 +107,11 @@ export const VERSION_ENV_VAR = 'DD_VERSION'
 export const ENVIRONMENT_ENV_VAR = 'DD_ENV'
 export const EXTRA_TAGS_ENV_VAR = 'DD_TAGS'
 export const CAPTURE_LAMBDA_PAYLOAD_ENV_VAR = 'DD_CAPTURE_LAMBDA_PAYLOAD'
+export const ENABLE_PROFILING_ENV_VAR = 'CORECLR_ENABLE_PROFILING'
+export const PROFILER_ENV_VAR = 'CORECLR_PROFILER'
+export const PROFILER_PATH_ENV_VAR = 'CORECLR_PROFILER_PATH'
+export const DOTNET_TRACER_HOME_ENV_VAR = 'DD_DOTNET_TRACER_HOME'
+export const INTEGRATIONS_ENV_VAR = 'DD_INTEGRATIONS'
```

Comment:
Update from Max: We actually don't need to apply this DD_INTEGRATIONS env var, so we can remove it throughout this PR